src/Numerics/DGMethods/DGModel.jl

```diff
@@ -905,3 +949,45 @@ function MPIStateArrays.MPIStateArray(dg::DGModel)
 
     return state_conservative
 end
+
+function contiguous_field_gradient!(
```

Comment:
A short docstring describing what this function is doing will be very nice.

Reply:
Done.